sns_ik_examples/src/ik_tests.cpp

```diff
@@ -67,21 +67,21 @@ double getDeltaWithLimits(double value, double desired_delta,
 
 bool in_vel_bounds(const KDL::JntArray& vel_values, const KDL::JntArray& vel_limits)
 {
-  for(size_t i; i < vel_limits.data.size(); i++){
-      if(vel_values(i) < -vel_limits(i)-1e-6 || vel_values(i) > vel_limits(i)+1e-6){
-          return false;
-      }
+  for (int i = 0; i < vel_limits.data.size(); i++) {
```

Comment:
Eigen matrix size() returns an int and not size_t which will make the compiler complain.